node/Makefile

```diff
@@ -610,13 +642,17 @@ sub-image-tar-windows-%:
 			--no-cache \
 			--build-arg=WINDOWS_VERSION=$* .
 
-# The calico-windows-upgrade cd is different because we do not build docker images directly.
```

Comment:
Docker

Reply:
Good point, this isn't needed here in the line removed.